lib/master.js

```diff
@@ -45,14 +47,14 @@ Master.prototype._run = function() {
     this._logger.log('info/service-runner', 'master(' + process.pid + ') initializing '
         + this.config.num_workers + ' workers');
 
-    cluster.on('exit', function(worker, code, signal) {
+    cluster.on('exit', function(worker) {
         if (!self._shuttingDown && !self._inRollingRestart) {
             var exitCode = worker.process.exitCode;
             var info = {
                 message: 'worker ' + worker.process.pid + ' died (' + exitCode + '), restarting.'
             };
             if (self.workerStatusMap[worker.process.pid]
-                    && self.workerStatusMap[worker.process.pid].status) {
+                && self.workerStatusMap[worker.process.pid].status) {
```

Comment:
Nit: let's re-indent it to make it clear this is part of the `if` clause, not the block.